src/shogun/classifier/mkl/MKL.cpp

```diff
@@ -271,23 +271,25 @@ void CMKL::register_params()
 	rho = 0;
 	lp_initialized = false;
 
-	SG_ADD(&svm, "svm", "wrapper svm");
+	SG_ADD(&svm, "svm", "wrapper svm", ParameterProperties::SETTING);
 	SG_ADD(&C_mkl, "C_mkl", "C mkl", ParameterProperties::HYPER);
-	SG_ADD(&mkl_norm, "mkl_norm", "norm used in mkl");
+	SG_ADD(&mkl_norm, "mkl_norm", "norm used in mkl", ParameterProperties::HYPER);
 	SG_ADD(&ent_lambda, "ent_lambda", "elastic net sparsity trade-off parameter",
 			ParameterProperties::HYPER);
 	SG_ADD(&mkl_block_norm, "mkl_block_norm", "mkl sparse trade-off parameter",
 			ParameterProperties::HYPER);
 
 	m_parameters->add_vector(&beta_local, &beta_local_size, "beta_local", "subkernel weights on L1 term of elastic net mkl");
-	watch_param("beta_local", &beta_local, &beta_local_size);
+	watch_param("beta_local", &beta_local, &beta_local_size,
+			AnyParameterProperties("subkernel weights on L1 term of elastic net mkl",
+					ParameterProperties::MODEL));
 
-	SG_ADD(&mkl_iterations, "mkl_iterations", "number of mkl steps");
-	SG_ADD(&mkl_epsilon, "mkl_epsilon", "mkl epsilon");
+	SG_ADD(&mkl_iterations, "mkl_iterations", "number of mkl steps", ParameterProperties::MODEL);
+	SG_ADD(&mkl_epsilon, "mkl_epsilon", "mkl epsilon", ParameterProperties::HYPER);
```

Comment:
@karlnapf epsilon values: HYPER or SETTING?

Reply:
I would make those setting

Reply:
Or hyper, but not model